pkg/helm/handlers/handler_test.go

```diff
@@ -187,3 +193,45 @@ func TestHelmHandlers_HandleHelmRenderManifest(t *testing.T) {
 		})
 	}
 }
+
+func TestHelmHandlers_HandleGetRelease(t *testing.T) {
+	tests := []struct {
+		name             string
+		expectedResponse string
+		release          *release.Release
+		error
+		httpStatusCode int
+	}{
+		{
+			name:             "Error occurred at listing releases",
```

Comment:
The test name should be updated, probably leftover from copy/pasting. 

This test is about getting a release, i.e. error occurring while doing it.